src/vscode-bicep/package.params.json

```diff
@@ -0,0 +1,27 @@
+{
+  "activationEvents": [
+    "onLanguage:bicep-params"
+  ],
+  "contributes": {
+    "languages": [
+      {
+        "id": "bicep-params",
+        "aliases": [
+          "Bicep Parameters",
+          "Bicep Params"
+        ],
+        "extensions": [
+          ".bicepparam"
+        ],
+        "configuration": "./syntaxes/language-configuration.json"
+      }
+    ],
+    "grammars": [
+      {
+        "language": "bicep-params",
+        "scopeName": "source.bicep",
+        "path": "./syntaxes/bicep.tmlanguage"
```

Comment:
Not in scope for this PR, but will bicep param files need a separate textmate grammar?

Reply:
I'm not sure. @anthony-c-martin, thoughts?